src/comp/comp.py

```diff
@@ -24,48 +24,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [h.name for h in humans if h.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [h.name for h in humans if h.name[-1] == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
-# whose name starts with any letter between 'C' and 'G' inclusive.
+# whose name starts with any letter between 'C' and 'G' inclusive. 103 - 107
 print("Starts between C and G, inclusive:")
-c = []
+c = [h.name for h in humans if h.name[0] in ("C", "D", "E", "F", "G")]
```

Comment:
A list looks like this: `[1, 2, 3, 4]`
A tuple: `(1, 2, 3, 4)`
A set: `{1, 2, 3, 4}`
A dict: `{1: "hi", 2: "bye" }`